app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -1,29 +1,73 @@
 <template>
-  <div class="widget-wrapper">
-    <WidgetHead :config="getWidgetHeadConfig" />
-    <WidgetBody />
-    <WidgetFooter />
+  <div>
+    <div v-if="isWidgetVisible" class="screen-selector">
+      <input-radio-group
+        name="widget-screen"
+        :items="widgetScreens"
+        :action="handleScreenChange"
+      />
+    </div>
+    <div v-if="isWidgetVisible" class="widget-wrapper">
+      <WidgetHead :config="getWidgetHeadConfig" />
+      <WidgetBody :config="getWidgetBodyConfig" />
+      <WidgetFooter :config="getWidgetFooterConfig" />
+      <div class="branding">
+        {{ $t('INBOX_MGMT.WIDGET_BUILDER.BRANDING_TEXT') }}
+      </div>
+    </div>
+    <div
+      class="widget-bubble"
+      :style="{
+        justifyContent: widgetBubblePosition === 'left' ? 'start' : 'end',
+      }"
+    >
+      <button
+        class="bubble"
+        :class="{
+          'bubble-close': isWidgetVisible,
+          'bubble-expanded':
+            !isWidgetVisible && widgetBubbleType === 'expanded_bubble',
+        }"
+        :style="{ background: color }"
+        @click="toggleWidget"
+      >
+        <img
+          v-if="!isWidgetVisible"
+          src="~dashboard/assets/images/bubble-logo.svg"
+          alt=""
+        />
+        <div>
+          {{
+            isWidgetVisible || widgetBubbleType === 'standard'
+              ? ' '
+              : widgetBubbleLauncherTitle
+          }}
+        </div>
+      </button>
+    </div>
   </div>
 </template>
 
 <script>
 import WidgetHead from './WidgetHead';
 import WidgetBody from './WidgetBody';
 import WidgetFooter from './WidgetFooter';
+import InputRadioGroup from '../../../routes/dashboard/settings/inbox/components/InputRadioGroup.vue';
 
 export default {
   name: 'Widget',
   components: {
     WidgetHead,
     WidgetBody,
     WidgetFooter,
+    InputRadioGroup,
   },
   props: {
     welcomeHeading: {
       type: String,
-      default: 'Hi There,',
+      default: 'Hi There!',
```

Comment:
i18n here. 

Also, we might not need to have a default here, we can just pass the i18n string from the parent component as a prop

Reply:
Done